app/services/form-repository.js

```diff
@@ -0,0 +1,91 @@
+import Service from '@ember/service';
+
+import { timeout } from 'ember-concurrency';
+import {
+  JSON_API_TYPE,
+  RESOURCE_CACHE_TIMEOUT,
+} from 'frontend-lmb/utils/constants';
+
+export default class FormRepositoryService extends Service {
+  async createFormInstance(instanceUri, definitionId, ttlCode) {
+    const result = await fetch(`/form-content/${definitionId}`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': JSON_API_TYPE,
+      },
+      body: JSON.stringify({
+        contentTtl: ttlCode,
+        instanceUri: instanceUri,
+      }),
+    });
+
+    await timeout(RESOURCE_CACHE_TIMEOUT);
+
+    return await this._handleCreateFetchhResult(result);
+  }
+
+  async updateFormInstance(
+    instanceId,
+    instanceUri,
+    definitionId,
+    ttlCode,
+    description
+  ) {
+    const result = await fetch(
+      `/form-content/${definitionId}/instances/${instanceId}`,
+      {
+        method: 'PUT',
+        headers: {
+          'Content-Type': JSON_API_TYPE,
+        },
+        body: JSON.stringify({
+          contentTtl: ttlCode,
+          instanceUri: instanceUri,
+          description: description,
+        }),
+      }
+    );
+
+    await timeout(RESOURCE_CACHE_TIMEOUT);
+
+    return await this._handleUpdateFetchhResult(result);
+  }
+
+  async _handleCreateFetchhResult(result) {
```

Comment:
Typos! `_handleCreateFetchhResult` and `_handleUpdateFetchhResult` have too many h'es
besides that, I would rename it to `handleCreateResponse` and `handleUpdateResponse`

Reply:
Is it validating something? Ill change it to `_handleCreateResult` and `_handleUpdateResult`